qiskit/algorithms/amplitude_estimators/ae.py

```diff
@@ -56,9 +61,10 @@ class AmplitudeEstimation(AmplitudeEstimator):
     def __init__(
         self,
         num_eval_qubits: int,
-        phase_estimation_circuit: Optional[QuantumCircuit] = None,
-        iqft: Optional[QuantumCircuit] = None,
-        quantum_instance: Optional[Union[QuantumInstance, Backend]] = None,
+        phase_estimation_circuit: None | QuantumCircuit = None,
+        iqft: None | QuantumCircuit = None,
+        quantum_instance: None | Union[QuantumInstance, Backend] = None,
```

Comment:
```suggestion
        quantum_instance: None | QuantumInstance | Backend = None,
```
Please check other places.